WordPressAuthenticatorTests/Logging/LoggingTests.m

```diff
@@ -9,6 +9,11 @@ @interface CaptureLogs : NSObject<WordPressLoggingDelegate>
 
 @end
 
+// We are leaving some protocol methods intentionally unimplemented to then test that calling them
+// will not cause a crash.
+//
+// See https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/720#issuecomment-1374952619
+#pragma GCC diagnostic ignored "-Wprotocol"
```

Comment:
🤦‍♂️ sorry, not sure why I wrote "gcc" in my previous comment. I think the right directive should be "clang".

Also, maybe we can add `#pragma clang diagnostic pop` and the end of the `CaptureLogs` implementation to stop ignoring the flag?

Reply:
No worries. I could have thought about it and checked.

However, that `#pragma` successfully disable the warning 🤔 

(I'm running out of time today so you might not see a commit to make the change after this comment)

Reply:
clang is compatible with gcc, so I guess it makes sense that the gcc directive works as well.

Reply:
Done. See [wordpress-mobile/WordPressAuthenticator-iOS@`8d7f99f` (#720)](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/720/commits/8d7f99f3b4aaa1c870395bcb1d268ebc63d5e1e6)